official-antistasi-community/A3-Antistasi
# Add 3CB Factions Boat nodes.

## What type of PR is this.
1. [x] Bug
2. [ ] Change
3. [ ] Enhancement

### What have you changed and why?
Information:
Added Logistic Nodes for:
UK3CB_CCM_I_Fishing_Boat, 4 Nodes in the Back
UK3CB_CCM_I_Fishing_Boat_VIV_FFV, 4 Nodes in the Back
UK3CB_CHD_I_Fishing_Boat_Zu23, 1 Node in the Front, it isnt much but atleast it can load a small Weapon box
UK3CB_CHD_I_Fishing_Boat_Zu23_front, 4 Nodes in the Back
UK3CB_CHC_C_Small_Boat_Wood, 2 Node in the Front ,should probably  disabled Mortars as the boat almost drowns when firing.
UK3CB_CHC_C_Small_Boat_Closed, 1 Node in the Front, it isnt much but atleast it can load a small Weapon box
UK3CB_CHC_C_Small_Boat_Open 1 Node in the Front, it isnt much but atleast it can load a small Weapon box

Deleted old missmerged Files


### Please specify which Issue this PR Resolves.
closes #1848 

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [ ] No
2. [x] Yes (Please provide further detail below.)
If Hakon doesent approve it there will be changes requiered :D
### How can the changes be tested?
Steps: 
Spawn Vehicles, load Boxes into Nodes, check positions for Clipping.
Also check for Blocked seats when Cargo is loaded:
Load Cargo, get into Seats and check for Clipping with Objects.
```sqf
vic = vehicle player;
moveOut player; player moveInCargo [vic, 0] //Change seat Position to see which one is used.
```
********************************************************
Notes:
